tests/5.0/target_teams_distribute/test_target_teams_distribute_reduction_multiply.c

```diff
@@ -0,0 +1,75 @@
+//===--- test_target_teams_distribute_reduction_multiply.c-------------------===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test uses the reduction clause on a target teams distribute directive,
+// testing that the variable in the reduction clause is properly reduced using
+// the multiply operator.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int test_multiply() {
+  int a[N];
+  int errors = 0;
+  int num_teams[N];
+  int warned = 0;
+  srand(1);
+
+  for (int x = 0; x < N; ++x) {
+    a[x] = 1 + (int) rand() / (double) RAND_MAX;
+    num_teams[x] = -x;
+  }
+
+  int result = 1;
+  int host_result;
+
+  for (int x = 0; x < N; x = x + 16) {
+    result = 1;
+#pragma omp target teams distribute reduction(*:result) map(to: a[0:N]) map(tofrom: result, num_teams[0:N])
+    for (int y = 0; y < 16; ++y) {
+      result *= a[x + y];
+      num_teams[x + y] = omp_get_num_teams();
+    }
+    host_result = 1;
+    for (int y = 0; y < 16; ++y) {
+      host_result *= a[x + y];
+    }
+    OMPVV_TEST_AND_SET_VERBOSE(errors, host_result != result);
+    OMPVV_INFOMSG("Device result is %d and host result is %d.", result, host_result);
+    if (host_result != result) {
+      break;
```

Comment:
This break will cause that the num_teams will not be filled completely, causing false errors later on in lines 53

Reply:
Fixed